src/FilterAdder.js

```diff
@@ -14,12 +14,12 @@ let FilterAdder = ({
   node,
   path,
   fields,
-  uniqueFields,
+  allowDuplicateFields,
```

Comment:
Well this _is_ a breaking change since `uniqueFields` is part of the public API

Reply:
Yeah I just reverted it back